en/guide/missions.md

```diff
@@ -190,11 +207,23 @@ The following synchronous methods are also available for checking mission progre
 
 If using a copter or VTOL vehicle then PX4 will automatically takeoff when it is armed and a mission is started (even without a takeoff mission item). For Fixed Wing vehicles the vehicle must be launched before starting a mission.
 
-At time of writing the Mission API does not provide takeoff, land or "return to launch" `MissionItems`. If required you can instead use the appropriate commands in the [Action](../api_reference/classdronecore_1_1_action.md) class.
+At time of writing the Mission API does not provide takeoff, land or "return to launch" `MissionItems`. If required you can instead use the appropriate commands in the [Action](../guide/taking_off_landing.md) class.
 
-<!-- when we have a topic on taking off and landing in the guide, link to that instead -->
 <!-- Update if we get new mission items -->
 
+## Downloading Missions
+
+Use [Mission::download_mission_async()](../api_reference/classdronecore_1_1_mission.md#classdronecore_1_1_mission_1a1bd15f508fe7da39b587a8e4d5e59ae2) to download a mission from the vehicle. The mission is downloaded as a vector of [MissionItem](../api_reference/classdronecore_1_1_mission_item.md) objects, that you can then view or manipulate as required.
+
+> **Note** Mission download will fail if the mission contains a command that is outside the [supported set](#supported_mission_commands). 
+> Missions created using *QGroundControl* are not guaranteed to successfully download! 
+
+The example below shows how to download a mission:
+
+```cpp
+// TBD

```

Comment:
@julianoes As per slack, appreciate your advice on best way to construct this example. Otherwise this PR ready for review.

Reply:
Fixed